src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -73,6 +75,17 @@ new Replay({
 
 <Note>Captured bodies will be truncated to 150k characters max.</Note>
 
+## Mutation Limits
+
+Session Replay works by recording incremental DOM changes that occur in your web application. Generally these changes occur in small batches and cause minimal overhead. However, it can be easy to overlook cases that will cause a large amount of DOM mutations to happen in a single update. An example is a custom dropdown component that has an unbounded list of options to render. This can negatively impact performance without Session Replay and its effects will be magnified with Session Replay enabled. In order to avoid this scenario, Session Replay will stop recording if it detects a large number of mutations (default: 10,000), which can be configured by setting `mutationLimit`. Additionally, we provide breadcrumbs in the replay to warn you when a large number of mutations are detected (750).
```

Comment:
@bruno-garcia can you help me write up something in the performance overhead section of docs that links to this?

Was thinking we find some links talking about virtualization of tables, etc to improve performance, as a guide/example of the performance impacts of this normally (without replays)